src/App.tsx

```diff
@@ -1,14 +1,65 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { FilterType } from './types/FilterType';
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/228150528-3cc5e1f2-19ab-43f5-a2f5-277bc3f8c89c.png)
Add all the files to your GitHub commit